src/components/settings/SettingsLMSTab/LMSConfigs/Degreed2Config.jsx

```diff
@@ -0,0 +1,160 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Button, Form, useToggle } from '@edx/paragon';
+import isEmpty from 'lodash/isEmpty';
+
+import { buttonBool, handleErrors } from '../LMSConfigPage';
+import LmsApiService from '../../../../data/services/LmsApiService';
+import { snakeCaseDict, urlValidation } from '../../../../utils';
+import ConfigError from '../ConfigError';
+import ConfigModal from '../ConfigModal';
+import { INVALID_LINK, INVALID_NAME, SUCCESS_LABEL } from '../../data/constants';
+
+const Degreed2Config = ({ id, onClick, existingData }) => {
```

Comment:
hmm if `id` is enterprise_customer_uuid we should probably just name it that? or `enterpriseCustomerUuid` in more camelCasey way